proxies/live/apiproxy/policies/OAuthV2.VerifyAccessToken.xml

```diff
@@ -1,3 +1,4 @@
 <OAuthV2 async="false" continueOnError="false" enabled="true" name="OauthV2.VerifyAccessToken">
   <Operation>VerifyAccessToken</Operation>
+  <Scope>urn:nhsd:apim:app:level3:e-referrals-service-api urn:nhsd:apim:user-nhs-id:aal3:e-referrals-service-api</Scope>
```

Comment:
Does this not allow for tokens created via the application-restricted route to be accepted as well as user restricted? I think only the "urn:nhsd:apim:user-nhs-id:aal3:e-referrals-service-api" scope should be being included here.